src/main/java/com/mentoring/ecommerce/adapter/in/web/request/ProductReq.java

```diff
@@ -0,0 +1,26 @@
+package com.mentoring.ecommerce.adapter.in.web.request;
+
+import com.sun.istack.internal.NotNull;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@NoArgsConstructor
+public class ProductReq {
+
+    private String id;
+
+    @NotNull
```

Comment:
There are various other annotations you can use to validate your fields, such as `@NotBlank` for strings, `@Min` and `@Max` for integers, longs, etc. Here are some: https://education.launchcode.org/java-web-development/chapters/spring-model-validation/validation-annotations.html